sql/src/main/java/io/crate/analyze/InsertFromValuesAnalyzer.java

```diff
@@ -292,6 +294,9 @@ private void addValues(DocTableRelation tableRelation,
             }
         }
 
+        // Validate Constraints
```

Comment:
I think the comment is a bit redundant? `ConstraintsValidator.validate` seems clear enough to me